apache-spark-master/run-master.sh

```diff
@@ -0,0 +1,3 @@
+/opt/apache-spark/sbin/start-master.sh
+
+tail -f /opt/apache-spark/logs/* 1>&2
```

Comment:
I've added this command just to keep the Docker container running. If you just do `start-master.sh`, the master starts... and the Docker container shuts down.